test/scenarios/release/release_test.go

```diff
@@ -5,22 +5,27 @@ package release
 import (
 	"testing"
 
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/dtversion"
 	"github.com/Dynatrace/dynatrace-operator/test/features/cloudnative/upgrade"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/operator"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/environment"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/tenant"
+	"golang.org/x/mod/semver"
 	"sigs.k8s.io/e2e-framework/pkg/env"
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
 var testEnv env.Environment
+var thresholdVersion, _ = dtversion.ToSemver("1.1.0")
+
+const usedVersion = "0.15.0"
```

Comment:
hmm... any reason why do we use old version for running e2e tests?

Reply:
We should bump this version as the test should just test the upgrade from previous to latest version

Reply:
☝️  that was my point as well

Reply:
Fair enough. I will bump it to 1.0.0 then I guess?